package/yast2-storage-ng.changes

```diff
@@ -1,3 +1,10 @@
+-------------------------------------------------------------------
+Wed Jul 25 21:41:48 CEST 2018 - aschnell@suse.com
+
+- use "Partition Table" instead of "Disk Label" in expert
+  partitioner (bsc#1070570)
+- 4.0.200
```

Comment:
If this change is only for master and not for SLE15SP0 (which makes sense, we should not change labels in MU with no good reason), then this is not following the versioning.
http://yastgithubio.readthedocs.io/en/latest/versioning/